Airports/EGVN/Runway.txt

```diff
@@ -1 +1 @@
-07  25  074 254 N051.44.45.930 W001.36.14.810 N051.45.13.950 W001.33.42.380
\ No newline at end of file
+07  25  073 254 N051.44.45.930 W001.36.14.810 N051.45.13.950 W001.33.42.380
```

Comment:
```suggestion
07  25  074 254 N051.44.45.930 W001.36.14.810 N051.45.13.950 W001.33.42.380
```
Although you are technically correct here, I'd just leave it as it was, otherwise the two runway headings at each end aren't 180 degrees out and that could cause some issues :)